src/sections/service-mesh-management/ServiceMesh.style.js

```diff
@@ -0,0 +1,96 @@
+import styled from "styled-components";
+
+const MeshSectionWrapper = styled.section`
+  margin: 1rem 0 5rem
+  position: relative;
+  overflow: hidden;
+  .title{ 
+    text-transform: uppercase;
+    color: #A0AAAA;
+  }
+  .side-text{
+    font-size: 34px;
+    font-weight: 400;
+    line-height: 44.2px;
+    color: ${props => props.theme.headingColor ? props.theme.headingColor : "#2C0075"};
+    margin: 0 0 24px 0;
+    span{
+        font-weight: 600;
+    }
+   }
+  .backBtn {
+        color: #00B39F;
+        cursor: pointer;
+        &:hover{
+            .icon{
+                transform: translate3d(5px, 0, 0);
+            }
+         }
+        line-height: 21px;
+        font-size: 21px;
+        margin-top: 21px;
+    }
+   .mesh-image{
+        height: 397px; 
+   }
+   .icon{
+       padding: 0px;
+       margin: 3px;
+       cursor: pointer;
+       vertical-align: middle;
+       }
+       
+   }
+   .description{
+       margin-bottom: 24px;
+       font-style: normal;
+       font-weight: 300;
+       font-size: 18px;
+       line-height: 27px;
+   }
+
+  @media only screen and (max-width: 480px) {
+    padding: 40px 0 60px 0;
+    .mesh-image{
+        display: none;
```

Comment:
Any specific reasons behind this?

Reply:
In the mobile view, the image should be visible or disappear?

Reply:
I think it should be visible.